packages/@dcl/ecs/src/engine/component.ts

```diff
@@ -46,6 +47,12 @@ export interface BaseComponent<T> {
    */
   getCrdtUpdates(): Iterable<CrdtMessageBody>
 
+  /**
+   * This function writes the whole state of the component into a ByteBuffer
+   * @public
+   */
+  dumpCrdtState(buffer: ByteBuffer): void
```

Comment:
```suggestion
  dumpCrdtStateToBuffer(buffer: ByteBuffer): void
```
It might exist a dumpToJson?

Reply:
good catch, i ll rename it in a separate PR 👍

Reply:
Done here: https://github.com/decentraland/js-sdk-toolchain/pull/455